submissions/oluwatosinOke/react/src/App.js

```diff
@@ -0,0 +1,120 @@
+import React, {useState, useEffect} from "react";
+import "./App.css";
+import 'bootstrap/dist/css/bootstrap.min.css';
+
+
+function Todo({ todo, index, completeTodo }) {
+  return (
+    <div className="todo"
+      style={{ textDecoration: todo.isCompleted ? "line-through" : "",color: todo.isCompleted ? "green" : ""}}>
+      {todo.text}
+      <div>
+        <label className='container'>
+        <input type="checkbox"  onClick={() => completeTodo(index)}/>
+        <span className="checkmark"></span>
+        </label>
+      </div>
+    </div>
+  );
+}
+
+function TodoForm({ addTodo }) {
+  const [value, setValue] = React.useState("");
+  const handleSubmit = e => {
+    e.preventDefault();
+    if (!value) return;
+    addTodo(value);
+    setValue("");
+  };
+  return (
+    <div className='todo-header'>
+      <h1>TODO</h1>
+      <div className='todo-create'>
+        <form onSubmit={handleSubmit}>
+          <input
+            placeholder="Create a new todo..."
+            type="text"
+            className="input"
+            value={value}
+            onChange={e => setValue(e.target.value)}/>
+          <div className='todo-button'>
+            <button type='submit'>+</button>
+          </div>
+        </form>
+      </div>
+    </div>
+  );
+}
+
+function App() {
+  const [todosRemaining,setTodosRemaining] = useState(0); 
+  const [todos, setTodos] = React.useState([
```

Comment:
@Tocyn You are not expect to have the list in state. You are meant to create the list in the browser and push the values in to the array here.